src/uct/api/uct.h

```diff
@@ -826,6 +826,15 @@ enum uct_md_mem_flags {
      */
     UCT_MD_MEM_ACCESS_LOCAL_WRITE   = UCS_BIT(9),
 
+    /**
+      * Register the memory region so its remote access key would likely be
+      * equal to remote access keys received from other peers, when compared
+      * with @ref uct_rkey_compare. This flag is best-effort. When remote access
```

Comment:
I'd suggest the following to solve docs build problem
```suggestion
      * with @a uct_rkey_compare. This flag is best-effort. When remote access
```

Reply:
fixed